### PR TITLE
Fixed save

### PR DESCRIPTION
**Pull request by popkinj**
The latest data transform broke the save functionality. Now fixed.
Made some minor updates to both forms.
Tested some behaviour with longer form responses.

### CODE REVIEW

**Comment by seeker25** [comment]
It seems to save fine, previous to this PR.

Updating the test files - looks good to me though.

File: fpo-api/api/pdf.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def transform_data_for_pdf(original_data):
         pass
 
     try:
-        data["hearingLocation"] = Location.objects.get(id=hearingLocationId).name
+        data["hearingLocationPDF"] = Location.objects.get(id=hearingLocationId).name
```
Comment (WadeBarnes) [inline fpo-api/api/pdf.py:64]: Is the variable change from `hearingLocation` to `hearingLocationPDF` necessary?

**Comment by WadeBarnes** [comment]
> It seems to save fine, previous to this PR.

Was for me too.

**Comment by popkinj** [comment]
Hey @WadeBarnes ... Just rolled back the change in question.

**Pull request merged**